server.go

```diff
@@ -883,6 +901,9 @@ func (s *Server) handleStream(t transport.ServerTransport, stream *transport.Str
 	method := sm[pos+1:]
 	srv, ok := s.m[service]
 	if !ok {
+		if handled := s.processUnknownStreamHandler(t, stream, trInfo); handled {
```

Comment:
I'm not against adding functions. But it seems this function is not necessary though...
And the boolean `handled` always equals to `s.opts.unknownStreamDesc != nil`